.gitignore

```diff
@@ -32,3 +32,7 @@ event_*.json
 my-*
 tags
 testbin/
+
+# don't include generated files from e2e tests
+test/e2e/data/operator-*
+test/e2e/output/ns-operator*
```

Comment:
I guess need include full test/e2e/output  (there could be be different prefixes)